### PR TITLE
fix jsonPathToArray bug and update snapshot

### PR DESCRIPTION
**Pull request by ankhyk**

### CODE REVIEW

File: regression/__snapshots__/validateExamplesRegression_latestOnly.test.ts.snap [hunk 1]
```diff
@@ -160,8 +160,8 @@ Array [
     "code": "INVALID_FORMAT",
     "exampleJsonPath": "$parameters.serviceMemberId",
     "examplePosition": Object {
-      "column": 17,
-      "line": 2,
+      "column": 24,
```
Comment (raych1) [inline regression/__snapshots__/validateExamplesRegression_latestOnly.test.ts.snap:163]: Did you sample check if column/line no are correct?

Comment (ankhyk) [reply regression/__snapshots__/validateExamplesRegression_latestOnly.test.ts.snap:163]: I checked the first few and they are all correct

**Comment by raych1** [comment]
can you add changelog?

**Review by raych1 — APPROVED** [review]

**Pull request merged**